workflow/dumping-workflows.rst

```diff
@@ -9,6 +9,7 @@ them as SVG or PNG images. First, install any of these free and open source
 applications needed to generate the images:
 
 * `Graphviz`_, provides the ``dot`` command;
+* `Mermaid CLI`, provides the ``mmdc``command;
```

Comment:
```suggestion
* `Mermaid CLI`_, provides the ``mmdc``command;
```

(the `` `...`_ `` syntax is used for links in reStructuredText)

This also requires adding this line to the bottom of the article:

```
.. _`Mermaid CLI`: https://github.com/mermaid-js/mermaid-cli
```